ssc/cmod_mhk_wave.cpp

```diff
@@ -524,7 +524,7 @@ class cm_mhk_wave : public compute_module
                 throw exec_error("mhk_wave", "Wave height and Energy period arrays of equal length must be assigned");
 
             ssc_number_t* energy_hourly = allocate("hourly_energy", number_records);
-            ssc_number_t* energy_hourly_gen = allocate("gen", number_records);
+            ssc_number_t* energy_hourly_gen = allocate("gen", number_records * 3);
```

Comment:
So, energy_hourly - energy_hourly_gen[i*3] ? Is energy_hourly_gen[number_records*3) used for plotting or other outputs?

Reply:
I need to make some changes here. To have the wave energy outputs be compatible with the Combine Cases functionality there either needs to be a 1-hour timestep output or unique data handling in the compare cases tool. Gen needs to be the kW output at each 3-hour timestamp to be consistent with the rest of the models and allow for the wave model to be used properly in tools such as reV.